platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -9,34 +9,70 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  Controls the scaling behavior of the circle when the map is pitched.
+ 
+ This property can be set to one of the following values:
```

Comment:
This isn’t a property, it’s an enumeration value. Enumeration values don’t need this extra text.